share/completions/dpkg-reconfigure.fish

```diff
@@ -8,7 +8,7 @@ complete -x -f -c dpkg-reconfigure -s h -l help     --description 'Display help'
 # General options
 complete -f -c dpkg-reconfigure -s f -l frontend -r -a "dialog readline noninteractive gnome kde editor web" --description 'Set configuration frontend'
 complete -f -c dpkg-reconfigure -s p -l priority -r -a "low medium high critical" --description 'Set priority threshold'
-complete -f -c dpkg-reconfigure -l default-priority --description 'Use default priority threshold'
+complete -f -c dpkg-reconfigure -l default-priority --description "Use current default ("(echo get debconf/priority | debconf-communicate | string match -r '\w*$')") priority threshold"
```

Comment:
debconf-communicate returns something like `0 medium` or `10 high`, where the first token seems to be a return (failure) code and the second token is what we are looking for. The string match selects the second token, thus `string trim` won't work. I didn't find a command line option to avoid returning the first token.

Splitting and indexing would be the most logical thing in my opinion, but I wanted to avoid introducing state (a variable) and something like
`echo "0 medium" | string split --index 2 " "`
does not exist.

However I just learned, that Fish can index subexpressions, which is even more flexible, i.e.
`echo (echo "0 medium" | string split " ")[2]`
works, although it looks a bit complex in this example.

But as a subexpression is already used, this actually gets easier
`complete -f -c dpkg-reconfigure -l default-priority --description "Use current default ("(echo get debconf/priority | debconf-communicate | string split ' ')[2]") priority threshold"`.

Should I create a new branch with that change?

P. S. Sorry for the need to cherry-pick. I haven't thought it through.